src/main/java/io/jenkins/plugins/gitlabbranchsource/GitLabSCMNavigator.java

```diff
@@ -99,10 +99,28 @@
      */
     private HashSet<String> navigatorProjects = new HashSet<>();
 
+    /**
+     * To store if project owner is group
+     */
+    private boolean isGroup;
```

Comment:
This should just be a method `isGroup()` which returns `return gitlabOwner instanceOf GitLabGroup`

Reply:
Wait, this cannot be this way because gitlabOwner is transient.